doc/vimb.1

```diff
@@ -1143,6 +1143,10 @@ selected.
 .B cursiv-font (string)
 The font family used as the default for content using cursive font.
 .TP
+.B dark-mode (bool)
+Whether to enable dark mode. Websites can use the `prefers-color-scheme' media
```

Comment:
Might be worth explicitly saying that this is GTK's dark mode. My initial assumption is that this turns on a custom style sheet in all sites.

Reply:
Though If I had read a little more closely I'd have seen the media query bit :)